app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class=container>
+  <h1 class="text-center my-4">Public Recipes</h1>
+  <% if @recipes.empty? %>
+    <p>No public recipes have been added yet</p>
+  <% end %>
+  <% @recipes.each do |recipe| %>
+    <% if recipe.public %>
+      <div class="border border-gray border-1 bg-body-tertiary py-4 px-5 mb-4 d-flex">
+        <div class="col-5">
+          <h4><%= recipe.name %></h4> 
+          <h6>By: <%= recipe.user.name %></h6> 
+        </div>
+        <div>
+          <p><%= "Total food items: #{recipe.total_quantity}"%></p>
+          <p><%= "Total price: $#{recipe.total_price}"%></p>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+  <div class="d-flex justify-content-center align-items-center mb-4">
+  </div>
```

Comment:
### Your Application

![image](https://user-images.githubusercontent.com/93039359/219694455-842ca348-7918-45f0-84ce-863d1321c770.png)

### Requirement
![image](https://user-images.githubusercontent.com/93039359/219694835-56858516-b1b1-4bd0-bce6-ec45927b8a65.png)


- [ ] It would be nice if we click on any of the public recipes and the application takes us to that recipe show/details page to show all the details of the recipe which is a requirement of your project :+1:

Reply:
resolved! ✔️